README.rst

```diff
@@ -0,0 +1,172 @@
+RPM List Builder
+================
+
+|Travis Build Status|
+
+.. |Travis Build Status| image:: https://travis-ci.org/sclorg/rpm-list-builder.svg?branch=master
+   :target: https://travis-ci.org/sclorg/rpm-list-builder
+
+RPM List Builder (rpmlb) helps you to build a list of defined RPM
+packages including Red Hat Software Collection (SCL) continually from
+the `recipe file <https://github.com/sclorg/rhscl-rebuild-recipes>`__.
+
+Features
+--------
+
+rpmlb ...
+
+-  Supports building a list of RPMs and SCL that is a extension of the
+   RPM packages.
+-  Supports several build types
+-  Mock (``mock``)
+-  Copr (``copr-cli``)
+-  Custom build by config file. You can customize the build with
+   ``rhpkg``, ``fedpkg``, ``koji``, ``brew``, ``dnf`` and etc.
+-  Supports several types to get packages by recipe file.
+-  Copy from local directory
+-  Download by ``rhpkg clone``.
+-  Custom download. You can customize the way with ``rhpkg``,
+   ``fedpkg``, and etc.
+-  Supports retry feature.
+-  Supports build by resume from any positon of the recipe file.
+
+Supported platforms
+-------------------
+
+-  Python 3.6 (Recommended), 3.5, 3.4
+
+Install
+-------
+
+Download
+~~~~~~~~
+
+Download files from the git repository.
+
+::
+
+    $ git clone REPO_URL
+
+Check release tags.
+
+::
+
+    $ git tag
+
+Check out to release version.
+
+::
+
+    $ git checkout vX.Y.Z
+
+Install to virtualenv environment
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You probably want to set up application on ``virtualenv``.
+
+Run below commmand to install ``virtualenv``. Python3 has ``virtualenv``
+as a standard library.
+
+::
+
+    $ pip3 install virtualenv
+
+Run below command to install application on virtualenv.
+
+::
+
+    $ make venv-install
+
+Run below command to check installed command.
+
+::
+
+    $ source venv/bin/activate
+    (venv) $ venv/bin/rpmlb -h
+    (venv) $ deactivate
+
+Install directly by pip
```

Comment:
Agree with @junaruga. Also it's hard to track changes made during format conversion.